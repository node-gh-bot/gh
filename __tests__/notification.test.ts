/**
 * Â© 2013 Liferay, Inc. <https://liferay.com> and Node GH contributors
 * (see file: CONTRIBUTORS)
 * SPDX-License-Identifier: BSD-3-Clause
 */

import { runCmd } from './runCmd'

describe('E2E: Notification Module Test', () => {
    it('List Notifications `gh nt`', done => {
        expect(runCmd('gh nt')).toMatchSnapshot()
        done()
    })
})
