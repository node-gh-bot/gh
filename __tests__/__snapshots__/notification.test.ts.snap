// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E: Notification Module Test List Notifications \`gh nt\` 1`] = `
"Listing activities on protoEvangelion/gh
protoEvangelion/gh
@protoEvangelion pushed 1 commit(s) to gh a year ago
@protoEvangelion created branch test-milestone at gh a year ago
@protoEvangelion pushed 1 commit(s) to gh a year ago
"
`;
